Makefile.am

```diff
@@ -85,6 +87,22 @@ BUILT_SOURCES += \
 	ir/ir-generated.cpp \
 	ir/gen-tree-macro.h
 
+######## P4 header files to install
+
+# FIXME -- should these be in $(pkgdatadir) instead of $(prefix)?  See configure.ac
+# FIXME -- and check-install-headers: below
+p4includedir = $(prefix)/p4include
+p4_14includedir = $(prefix)/p4_14include
+p4include_HEADERS += $(srcdir)/p4include/core.p4 $(srcdir)/p4include/v1model.p4
+
+#### For testing, install headers in the build directory
+
+check-install-headers:
+	@$(MAKE) install-data prefix=.
```

Comment:
Possibly, though I'm not sure under what circumstances $(builddir) could be anything other than .